milestone2.ts

```diff
@@ -164,9 +173,17 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  let sloOldPerson = data.find((vol) => (vol.city == "SLO" && vol.age > 40));
+
+  if (sloOldPerson == undefined) {
+    return undefined;
+  }
+
+  return Object.assign(sloOldPerson, {position: "staff"})
```

Comment:
I'm not sure why you used Object.assign here when we showed how to update an array using the spread operator just above.. IN THE FUTURE DO NOT USE Object.assign() WITHOUT FIRST MAKING A COPY. In this example, Object.assign is altering the input data when it is updating the object. We never want to alter the input data to our functions as we do not know if this will impact data in other places. Rather, we always want to make a copy. If you were to use the spread operator instead, that would return a copy of the data with the updated fields which is much safer and better practice. I suspect you got lucky that this was the last test in the page, otherwise your alterations of the main data could have caused problems.